### PR TITLE
Added blog post host devices

### PR DESCRIPTION
**Pull request by jordigilh**
Added access to host devices related documentation:
* Blog post
* Documentation

Pending on https://github.com/project-flotta/flotta-device-worker/pull/212 to be merged first.

### CODE REVIEW

File: _posts/2022-07-22-accessing-host-devices-in-flotta.markdown [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+drwxrwxrwt. 1 root root 4096 Jul 22 15:04 .
+```
+
+There we have the 10 images inside the container. As an exercise, you can enhance this example by adding a data sync path to upload the images to a remote S3 storage using the data transfer capabilities of the Flotta agent. This way you won't need to sneak into the VM like I did to see that the images were created.
+
+## Conclusion ##
+
+Flotta is gradually improving the support of workloads that can run on devices without compromising the security of the device. With Flotta it is possible to run workloads that generate and consume data and are capable of synchronizing with a remote compatible S3 storage, as well as workloads that require access to the host mounted devices, such as webcams or sensors.
+
+There is still room for improvement in the usability with the ability to configure the supplementary groups without having to remote to the device each time. We hope in the next releases we can provide a mechanism to configure the host in these areas in a declarative manner. Feel free to drop suggestions or enhancements to the [project](https://github.com/project-flotta/) in the github repository!.
```
Comment (masayag) [inline _posts/2022-07-22-accessing-host-devices-in-flotta.markdown:185]: nit: maybe specific link to issues: https://github.com/project-flotta/flotta-operator/issues

File: _posts/2022-07-22-accessing-host-devices-in-flotta.markdown [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+
+## Notes ##
+
+In the case of my webcam, I was able to add the specific rules to SELinux to allow the `continer_t` type to access the `v4l_device_t` type, which is the label type that is used by `/dev/video2`:
```
Comment (masayag) [inline _posts/2022-07-22-accessing-host-devices-in-flotta.markdown:189]: s/continer_t/container_t

File: _posts/2022-07-22-accessing-host-devices-in-flotta.markdown [hunk 3]
```diff
@@ -0,0 +1,322 @@
...
+[
+  {
+    "lastTransitionTime": "2022-07-22T15:02:28Z",
+    "name": "webcam",
+    "phase": "Running"
+  },
+  {
+    "lastTransitionTime": "2022-07-22T15:31:28Z",
+    "name": "webcam-nosecuritycontext",
+    "phase": "Running"
```
Comment (masayag) [inline _posts/2022-07-22-accessing-host-devices-in-flotta.markdown:299]: I'm surprised both workloads could run at the same time using the same device.

File: uml/e66519fc27225a6f5fab4d4af6c6d64e.uml [hunk 4]
```diff
@@ -0,0 +1,25 @@
+@startuml
```
Comment (masayag) [inline uml/e66519fc27225a6f5fab4d4af6c6d64e.uml:1]: should this file be part of this blog post?

File: _posts/2022-07-22-accessing-host-devices-in-flotta.markdown (hunk 1 above)
Comment (jordigilh) [reply _posts/2022-07-22-accessing-host-devices-in-flotta.markdown:185]: good idea. thanks

File: _posts/2022-07-22-accessing-host-devices-in-flotta.markdown (hunk 3 above)
Comment (jordigilh) [reply _posts/2022-07-22-accessing-host-devices-in-flotta.markdown:299]: well the first one was done with capturing the screenshots, so I guess there was no race condition in attempting to read/write from the device.

File: uml/e66519fc27225a6f5fab4d4af6c6d64e.uml (hunk 4 above)
Comment (jordigilh) [reply uml/e66519fc27225a6f5fab4d4af6c6d64e.uml:1]: nope, I'll remove it :smile:

**Review by masayag — APPROVED** [review]

**Pull request merged**